app/models/service_order.rb

```diff
@@ -5,14 +5,15 @@ class ServiceOrder < ActiveRecord::Base
 
   belongs_to :tenant
   belongs_to :user
-  has_many :miq_requests, :dependent => :nullify
+  has_many :miq_requests
```

Comment:
@gmcculloug 
The :nullify prevents the destroy_unprocessed_requests from doing the miq_requests.destroy_all because the miq_request service_id is nil.  I have a test that does the destroy, then checks that the service_order and its miq_requests have been deleted. This test fails if :dependent => :nullify is used.
Please advise.